example/ios/Podfile.lock

```diff
@@ -577,6 +575,6 @@ SPEC CHECKSUMS:
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   Yoga: 86fed2e4d425ee4c6eab3813ba1791101ee153c6
 
-PODFILE CHECKSUM: 2978050834ea11b0c4d563dcf828a133feaa5779
+PODFILE CHECKSUM: 8240754c24c60b9eb062f31249323bce32a16dcc
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.15.2
```

Comment:
Tell me if you want to keep cocoapods 1.12.0

Reply:
Did you run this and test that iOS builds? I don't know what the min target on PinCache is.

Reply:
Also, do you know how we could enable this to work with the bridgeless mode in RN 0.74? Would be awesome to have this working on new arch.

Reply:
Haven't tested on iOS 12, but documentation on PinCache https://github.com/pinterest/PINCache?tab=readme-ov-file#requirements say it's compatible with iOS 12+.

About bridgeless mode, I'll need to check, but it takes time and for now I don't have enough to migrate this lib. I can try to look at some time

Reply:
Cool. I appreciate it. I'm going to start converting some of our libraries, so I thought I'd ask!